ROADMAP.md

```diff
@@ -0,0 +1,27 @@
+# Roadmap
+
+This roadmap lists the main features and their statuses on the Kuzzle open-source core project, and all its associated satellite repositories (SDKs, plugins, documentation).
+
+
+## Planned
+
+* Q2 2016: allow users to have multiple profiles
+* Q2 2016: iOS SDK
+* Q2 2016: Load Balancing of multiple Kuzzle servers
+* Q3 2016: Syslog capabilities
+* Q3 2016: C SDK + wrappers to different languages, using SWIG (C#, Java, PHP, Python, Ruby, ...)
+* Q3 2016: Plugins configuration through the Kuzzle Back-Office
+
+
+## Won't do
+
+*This section is currently empty*
+
+## Done
+
+* Plugins configuration through the command-line interface (see [PR #253](https://github.com/kuzzleio/kuzzle/pull/253))
+* Added administrative functions to the command-line interface (see [PR #208](https://github.com/kuzzleio/kuzzle/pull/208))
```

Comment:
Inconsistency in tense.
Should be something like "more administrative functions..."